plugins/tsne.cc

```diff
@@ -379,34 +379,45 @@ TsneEmbedConfigDescription()
 /* TSNE EMBED ROW                                                            */
 /*****************************************************************************/
 
+DEFINE_STRUCTURE_DESCRIPTION(TsneInput);
+
+TsneInputDescription::TsneInputDescription()
+{
+    addField("embedding", &TsneInput::embedding,
+             "Undocumented.");
```

Comment:
That is a separate DOD; especially since the function is disabled there was no point in writing the doc.